[2021-08-16 15:47:21,382] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_complex.create_entry_gcs_result2 2021-08-16T15:43:47.414310+00:00 [queued]>
[2021-08-16 15:47:21,604] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_complex.create_entry_gcs_result2 2021-08-16T15:43:47.414310+00:00 [queued]>
[2021-08-16 15:47:21,623] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-08-16 15:47:21,630] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-08-16 15:47:21,633] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-08-16 15:47:21,719] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): create_entry_gcs_result2> on 2021-08-16T15:43:47.414310+00:00
[2021-08-16 15:47:21,811] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_complex', 'create_entry_gcs_result2', '2021-08-16T15:43:47.414310+00:00', '--job-id', '32', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_complex.py', '--cfg-path', '/tmp/tmp2khe2kbq', '--error-file', '/tmp/tmpcfjgho8n']
[2021-08-16 15:47:21,742] {standard_task_runner.py:52} INFO - Started process 383 to run task
[2021-08-16 15:47:21,837] {standard_task_runner.py:77} INFO - Job 32: Subtask create_entry_gcs_result2
[2021-08-16 15:47:22,181] {logging_mixin.py:104} INFO - Running <TaskInstance: example_complex.create_entry_gcs_result2 2021-08-16T15:43:47.414310+00:00 [running]> on host 8f195762143d
[2021-08-16 15:47:22,501] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_complex
AIRFLOW_CTX_TASK_ID=create_entry_gcs_result2
AIRFLOW_CTX_EXECUTION_DATE=2021-08-16T15:43:47.414310+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-16T15:43:47.414310+00:00
[2021-08-16 15:47:22,509] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-08-16 15:47:22,522] {bash.py:158} INFO - Running command: echo create_entry_gcs_result2
[2021-08-16 15:47:22,616] {bash.py:169} INFO - Output:
[2021-08-16 15:47:22,627] {bash.py:173} INFO - create_entry_gcs_result2
[2021-08-16 15:47:22,633] {bash.py:177} INFO - Command exited with return code 0
[2021-08-16 15:47:22,715] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=create_entry_gcs_result2, execution_date=20210816T154347, start_date=20210816T154721, end_date=20210816T154722
[2021-08-16 15:47:22,878] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-16 15:47:22,945] {local_task_job.py:146} INFO - Task exited with return code 0
